scripts/rollback-latest-deployment.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+environment=$1
+
+set -euo pipefail
+
+if [[ "$environment" != "staging" && "$environment" != "production" ]]; then
+  echo "Invalid environment '$environment'. Allowed values are 'staging' or 'production'."
+  exit 1
+fi
+
+export AWS_PROFILE=$environment
+
+echo "Retrieving previous application revision to deploy"
+
+latest_deployment_identifier=$(
+  aws deploy list-deployments \
+    --application-name AppECS-Forms-form-viewer \
+    --deployment-group-name DgpECS-Forms-form-viewer \
+    --max-items 1 \
+    --output json \
+    | jq -r ".deployments[0]"
+)
+
+previous_revision_sha256=$(
+  aws deploy get-deployment \
+    --deployment-id $latest_deployment_identifier \
+    --query "deploymentInfo.previousRevision.appSpecContent.sha256" \
+    --output text
+)
+
+echo "Create deployment using previous revision ($previous_revision_sha256)"
+
+new_deployment_identifier=$(
+  aws deploy create-deployment \
+    --application-name AppECS-Forms-form-viewer \
+    --deployment-group-name DgpECS-Forms-form-viewer \
+    --revision revisionType=AppSpecContent,appSpecContent={sha256=a72b5abc6eb1a424b958104085eec8a6fa0bce28ee32d307204dd3c125a8ce8b} \
```

Comment:
should the `appSpecContent` here be referencing the `$previous_revision_sha256` instead of being hardcoded?

Reply:
Good catch!